test/conformance/tests/sriov_operator.go

```diff
@@ -834,15 +850,71 @@ var _ = Describe("[sriov] operator", func() {
 
 		Describe("Configuration", func() {
 
+			Context("Create vfio-pci node policy", func() {
+				It("Should be possible to create a vfio-pci resource", func() {
+					if discovery.Enabled() {
+						Skip("Test unsuitable to be run in discovery mode")
+					}
+					if vfioNode == "" {
+						Skip("skip test as no vfio-pci capable PF was found")
+					}
+					By("creating a vfio-pci node policy")
+					resourceName := "testvfio"
+					config := &sriovv1.SriovNetworkNodePolicy{
```

Comment:
lets create a function for vfio

and lets use this https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/test/util/network/network.go#L52 as the base function like we have in the podDefine

Reply:
Won't it be a problem using that function as a base function? Because it applies the policy before returning it, meaning that if we redefine after it will apply twice.

Reply:
yep I want to create a new function just like the podDefine and this function will call it and also apply it to maintain the current path